### ISSUE TITLE
Missing config.yaml

### ISSUE THREAD

**Issue by radnonymous** [OpenTechStrategies/ots-bookkeeping#1]
https://github.com/OpenTechStrategies/ots-bookkeeping/blob/038445b8d7f5812b226594862b8a69f8bbae6006/threadfin/reconcile/reconcile.py#L227

This line references a file I can't find in the repository. Is the file missing, or is this the config file referenced in fcebca19c45c7015332d196c3adf5759c744b686 that you are trying to move away from?

**Comment by jvasile** [comment]
I grepped for config and see it is referenced in convert.py and reconcile.py.   Convert.py has it as a commandline option and then doesn't use it.  We should probably remove that option since we obviously don't need it.

Reconcile.py does appear to use config.yaml, and we have one on our system (we call it reconcile.yaml, but pass it in via --config in our standard reconcile runner script).  Mine is below.  You'll see I have a template dir, some accounts, some personal finance stuff, etc.

    template_dir: "/usr/local/src/ots-bookkeeping/threadfin/reconcile/templates"

    accounts: [
	{'name': 'main',
	'ledger_file': "/home/james/OTS/finances/main.beancount",
	},

	{'name': 'chase',
	'ledger_file': "/home/james/OTS/finances/statements/all.beancount",
	},

	{'name': 'james_chase',
	'ledger_file': "/home/james/Documents/Finances/statements/all.beancount",
	},

	{'name': 'aadvantage',
	'ledger_file': "/home/james/Sync/family/documents/finance/aadvantage/statements/all.beancount"
	},

	{'name': 'capitalone',
	'ledger_file': "/home/james/Sync/family/documents/finance/capitalone/statements/all.beancount"
	}
    ]


    reconcile: [
	{ 'chase':['Assets:Checking'],
	'main': ['Assets:Checking', 'Assets:Guaranteed-Payments']
	},
    ]

So, yes, perhaps it would be good to move away from the config file, but it looks like I didn't find a better place to specify some things.  And there is no config file in the repo because the values are so local.  The right answers here are probably to 1) remove the --config option from convert.py and 2) add a sample config.yaml file.  Feel free to take a swing at either of those!

**Comment by radnonymous** [comment]
For 2), the sample config.yaml would need to list accounts from sample beancount files and bank statements, correct? So making this change will also need to add sample files to make the config file work, right? Or is the intent for this config file to be a placeholder that the user will edit to match to their local values?

**Comment by jvasile** [comment]
You're right that samples would be the best way to go.  That starts to imply a larger task, which is to make a full set of sample files.

**Comment by radnonymous** [comment]
Where in the repository would be the most logical place to put the config and sample files?
